src/barril/units/unit_database.py

```diff
@@ -16,6 +17,20 @@
     "UnitsError",
 ]
 
+LEGACY_REPRESENTATIONS = ["1000ft3", "1000m3", "M(ft3)", "M(m3)", "k(ft3)"]
+CURRENT_REPRESENTATIONS = ["Mcf", "Mm3", "MMcf", "MMm3", "Mcf"]
+LEGACY_TO_CURRENT = dict(zip(LEGACY_REPRESENTATIONS, CURRENT_REPRESENTATIONS))
```

Comment:
Why three variables, if in the end you only use the latter? I suggest to only have `LEGACY_TO_CURRENT`, and create it explicitly. It also doesn't seem to be used as a dict, only as a list of pairs given that you only do a linear search on it. Also make it private. So:

```python
_LEGACY_TO_CURRENT = {
    ("1000ft3", "Mcf"),
    ("1000m3", "Mm3"),
    ("M(ft3)", "MMcf"),
    ("M(m3)", "MMm3"),
    ("k(ft3)", "Mcf"),
}
```